README.md

```diff
@@ -31,15 +31,16 @@ For an existing Angular CLI application,
 
 1. **Install Fundamental-NGX.**
 
-    `ng add fundamental-ngx`
-    
-    *If you do not use the Angular CLI or if this command does not work for you, please see the [full installation guide](https://github.com/SAP/fundamental-ngx/wiki/Full-Installation-Guide).*
+    `ng add @fundamental-ngx/core`
+
+    _If you do not use the Angular CLI or if this command does not work for you, please see the [full installation guide](https://github.com/SAP/fundamental-ngx/wiki/Full-Installation-Guide)._
```

Comment:
we need to update the full installation guide and explain the latest changes too

Reply:
done